ape/ape.S

```diff
@@ -540,11 +540,32 @@ apesh:	.ascii	"'\n#'\"\n"			# sixth edition shebang
 //	modify the binary to follow the local system's convention.
 //	There isn't a one-size-fits-all approach for this, thus we
 //	present two choices.
+	.ascii	"o=\"$(command -v \"$0\")\"\n"
+//	Try to use a system-wide APE loader.
+	.ascii	"type ape-loader >/dev/null 2>&1 && "
+	.ascii	"exec ape-loader \"$0\" \"$@\"\n"
+#ifdef APE_LOADER
+//	There is no system-wide APE loader, but there is one
```

Comment:
1. I would recommend having this only happen for the `ape-no-modify-self.o` build.
2. You might want to check `$(uname -s) = Linux`

Reply:
1.
I was in the impression that, if a system-wide ape-loader exist, it is always the right thing to do.
The copy of the embedded loader is done only if `APE_LOADER` is defined, which, I expect, happens only for `ape-no-modify-self.o`.

I need a separate macro, as `APE_LOADER` gives the path of the loader ELF.

2.
I just copied the test for Linux/MacOS which was already here. The check for the existence of a directory is faster than calling `uname` in a subshell.